docs/content/modeling/user-groups.mdx

```diff
@@ -78,34 +77,34 @@ In addition, you will need to know the following:
 
 ### <ProductName format={ProductNameFormat.ShortForm}/> Concepts
 
-- A <ProductConcept section="what-is-a-type" linkName="Type" />: a class of objects that have similar characteristics
-- A <ProductConcept section="what-is-a-user" linkName="User" />: an entity in the system that can be related to an object
-- A <ProductConcept section="what-is-a-relation" linkName="Relation" />: is a string defined in the type definition of an authorization model that defines the possibility of a relationship between an object of the same type as the type definition and a user in the system
-- An <ProductConcept section="what-is-an-object" linkName="Object" />: represents an entity in the system. Users' relationships to it can be define through relationship tuples and the authorization model
-- A <ProductConcept section="what-is-a-relationship-tuple" linkName="Relationship Tuple" />: a grouping consisting of a user, a relation and an object stored in <ProductName format={ProductNameFormat.ShortForm}/>
+- A <ProductConcept section="what-is-a-type" linkName="Type" />: a class of objects that have similar characteristics.
+- A <ProductConcept section="what-is-a-user" linkName="User" />: an entity in the system that can be related to an object.
+- A <ProductConcept section="what-is-a-relation" linkName="Relation" />: a string defined in the type definition of an authorization model that defines the possibility of a relationship between an object of the same type as the type definition and a user in the system.
+- An <ProductConcept section="what-is-an-object" linkName="Object" />: represents an entity in the system. Users' relationships to it can be defined with relationship tuples and the authorization model.
+- A <ProductConcept section="what-is-a-relationship-tuple" linkName="Relationship Tuple" />: a grouping consisting of a user, a relation and an object stored in <ProductName format={ProductNameFormat.ShortForm}/>.
 
 </details>
 
 <Playground />
 
 ## Step By Step
 
-As we develop our application, we might encounter use cases where a group of users have a certain role or permission on an object. For example, `members` of a certain `team` might have an `editor` relation to a certain `document`.
+You will encounter use cases where a group of users have a certain role on or permission to an object. For example, `members` of a certain `team` could have an `editor` relation to a certain `document`.
 
-In order to represent this in <ProductName format={ProductNameFormat.ShortForm}/>, we need:
+To represent this in <ProductName format={ProductNameFormat.ShortForm}/>, we need to:
 
-1. Introduce the concept of a `team` to the authorization model
-2. Add _users_ as `members` to the `team`
-3. Assign the `team` members a relation to an _object_
-4. Checking an individual member's access to the _object_
+1. Introduce the concept of a `team` to the authorization model.
```

Comment:
Just a thought -- but if each of these steps refers to a subheading below, should we link to them directly from this list?

Reply:
Makes sense -  we wanted to do that when we first wrote this but having the whole sentence be a link is not great. But there's not harm in adding a [→](...) at the end of each point

Reply:
Pushed changes to add section links using arrows as suggested.